packages/suite/src/utils/suite/env.ts

```diff
@@ -164,3 +164,11 @@ export const watchOsTheme = (callback: (theme: SuiteThemeVariant) => void) => {
     query?.addEventListener('change', onThemeChange);
     return () => query?.removeEventListener('change', onThemeChange);
 };
+
+export const UNFILTERED_HOSTNAME = 'passthrough';
```

Comment:
This is an interesting solution, add at least a comment here to link it with `request-filter`

Reply:
Agree, but we may have came up with different solution with @szymonlesisz (manually allow any request in electron popup instead of just notifying that it was blocked), so this might change.

Reply:
Sounds better!